components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -129,11 +130,13 @@ open class GleanInternalAPI internal constructor () {
         // If Glean is being initialized from a test, then we want to skip running the
         // startupCheck() as it can cause some intermittent test failures due to multi-
         // threaded race conditions.
-        @Suppress("EXPERIMENTAL_API_USAGE")
```

Comment:
Let's leave this here and add another suppression: this would make it easier to remove/reconsider them if needed.